vllm/v1/core/sched/interface.py

```diff
@@ -4,6 +4,11 @@
 from collections.abc import Iterable
 from typing import TYPE_CHECKING, Optional
 
+from vllm.config import VllmConfig
+from vllm.config.vllm import KVCacheConfig
```

Comment:
Should this not be imported from where it is defined?

```suggestion
from vllm.v1.kv_cache_interface import KVCacheConfig
```